app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,25 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def handle_invalid_logged_out_form
+      return if current_user
```

Comment:
There are already existing specs for the case where a user is logged in but provides a bad request

The problem I was trying to fix was behavior and analytics specifically for logged out users (because our processes for persisting data and redirecting was too fragile) so I wanted to skip that entirely if we know ahead of time that the requests will fail.